platform/darwin/src/MGLGeoJSONSource.mm

```diff
@@ -49,9 +50,20 @@ - (instancetype)initWithIdentifier:(NSString *)identifier URL:(NSURL *)url optio
     return self;
 }
 
+- (instancetype)initWithIdentifier:(NSString *)identifier feature:(id<MGLFeature>)feature options:(NSDictionary<MGLGeoJSONSourceOption,id> *)options
```

Comment:
Mostly not to break the API. Should we rename it to `-[MGLGeoJSONSource initWithIdentifier:shape:options:]` or keep `feature:` and `features:` while adding `shape:` ?